public/modules/custom/grants_handler/src/Plugin/Block/ServicePageAuthBlock.php

```diff
@@ -94,28 +59,23 @@ public function __construct(
     array $configuration,
     $pluginId,
     $pluginDefinition,
-    HelsinkiProfiiliUserData $helfiHelsinkiProfiili,
-    CurrentRouteMatch $routeMatch,
-    AccountProxy $user,
-    ServicePageBlockService $servicePageBlockService,
-    ApplicationStatusService $applicationStatusService,
+    protected HelsinkiProfiiliUserData $helfiHelsinkiProfiili,
+    protected CurrentRouteMatch $routeMatch,
+    protected AccountProxy $currentUser,
+    protected ServicePageBlockService $servicePageBlockService,
+    protected ApplicationStatusService $applicationStatusService,
   ) {
     parent::__construct($configuration, $pluginId, $pluginDefinition);
-    $this->helfiHelsinkiProfiili = $helfiHelsinkiProfiili;
-    $this->routeMatch = $routeMatch;
-    $this->currentUser = $user;
-    $this->servicePageBlockService = $servicePageBlockService;
-    $this->applicationStatusService = $applicationStatusService;
   }
 
   /**
    * {@inheritdoc}
    */
-  public static function create(ContainerInterface $container, array $configuration, $pluginId, $pluginDefinition) {
-    return new static(
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition): static {
```

Comment:
The return type hit (`static`) does not match with the actual return type (`self`)